src/go/rpk/pkg/cli/cmd/group/seek.go

```diff
@@ -0,0 +1,268 @@
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/config"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/kafka"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/out"
+)
+
+func newSeekCommand(fs afero.Fs) *cobra.Command {
+	var (
+		to             string
+		toGroup        string
+		toFile         string
+		topics         []string
+		allowNewTopics bool
+	)
+
+	cmd := &cobra.Command{
+		Use:   "seek [GROUP] --to (start|end|timestamp) --to-group ... --topics ...",
+		Short: "Modify a group's current offsets.",
+		Long: `Modify a group's current offsets.
+
+This command allows you to modify a group's offsets. Sometimes, you may need to
+rewind a group if you had a mistaken deploy, or fast-forward a group if it is
+falling behind on messages that can be skipped.
+
+The --to option allows you to seek to the start of partitions, end of
+partitions, or after a specific timestamp. The default is to reset any topic
+previously committed. Using --topics allows to you set the commits for only the
+specified topics; all other commits will remain untouched. Topics with no
+commits will not be committed unless specified with --allow-new-topics.
+
+The --to-group option allows you to seek to commits that are in another group.
+This is a merging operation: if g1 is consuming topics A and B, and g2 is
+consuming only topic B, "rpk group seek g1 --to-group g2" will update g1's
+commits for topic B only. The --topics flag can be used to further narrow which
+topics are updated.
+
+The --to-file option allows to seek to offsets specified in a text file with
+the following format:
+    [TOPIC] [PARTITION] [OFFSET]
+    [TOPIC] [PARTITION] [OFFSET]
+    ...
+Each line contains the topic, the partition, and the offset to seek to.
+
+The --to, --to-group, and --to-file options are mutually exclusive. If you are
+not authorized to describe or read some topics used in a group, you will not be
+able to modify offsets for those topics.
+
+EXAMPLES
+
+Seek group G to June 1st, 2021:
+    rpk group seek g --to 1622505600
+    or, rpk group seek g --to 1622505600000
+    or, rpk group seek g --to 1622505600000000000
+Seek group X to the commits of group Y:
+    rpk group seek X --to-group Y
+Seek group G's topics foo, bar, and biz to the end:
+    rpk group seek G --to end --topics foo,bar,biz
+Seek group G to the beginning of a topic it was not previously consuming:
+    rpk group seek G --to start --topics foo --allow-new-topics
+`,
+		Args: cobra.ExactArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			p := config.ParamsFromCommand(cmd)
+			cfg, err := p.Load(fs)
+			out.MaybeDie(err, "unable to load config: %v", err)
+
+			adm, err := kafka.NewAdmin(fs, p, cfg)
+			out.MaybeDie(err, "unable to initialize kafka client: %v", err)
+			defer adm.Close()
+
+			var n int
+			for _, f := range []string{to, toGroup, toFile} {
+				if f != "" {
+					n++
+				}
+			}
+			switch {
+			case n == 0:
+				out.Die("Missing specify one --to flag.")
```

Comment:
```suggestion
				out.Die("Missing one --to flag.")
```

Reply:
Whoops, I meant this to be `Must specify one --to flag`. I think that's a bit clearer than `Missing one`